README.md

```diff
@@ -121,7 +121,7 @@ vars:
 ```  
 
 #### Change the build schema
-By default, this package builds the SAP staging models within a schema titled (`<target_schema>` + `_sap_source`) in your destination. If this is not where you would like your sap staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
+By default, this package builds the SAP staging models within a schema titled (`<target_schema>` + `sap`) in your destination. If this is not where you would like your sap staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
```

Comment:
This is missing the callout for where the staging models are materialized. Can you add that relevant part in.

Reply:
thanks Joe, updated